radis/test/lbl/test_loader.py

```diff
@@ -86,9 +87,37 @@ def test_retrieve_from_database(
         rmtree(temp_database_name)
 
 
+def test_files_from_regex(plot=True, verbose=True, warnings=True, *args, **kwargs):
```

Comment:
good to have a full test, but you should also have a small test in `misc/test_utils.py` [new file probably] that imports an tests the `get_files_from_regex()` function alone

Reply:
Added the test in a new file.